### ISSUE TITLE
Add Fan node device?

### ISSUE THREAD

**Issue by umibozu83** [sammachin/node-red-matter-bridge#31]
Is it possible to add fan control in the future?

**Comment by sammachin** [comment]
Yes good comment, the Fan Control cluster is quite complex in terms of options, so I'll need to do some testing to see which features are actually exposed by the ecosystems

**Comment by sammachin** [comment]
added in 0.12, feedback welcome
